ui/app/templates/components/namespace-picker.hbs

```diff
@@ -42,18 +42,22 @@
         {{/if}}
 
         <div class="is-mobile level-left">
-          {{#unless this.isUserRootNamespace}}
+          {{#if this.isUserRootNamespace}}
+            <h5 class="list-header">Namespaces</h5>
+          {{else}}
             <NamespaceLink
               @targetNamespace={{or
                 (object-at (dec 2 this.menuLeaves.length) this.lastMenuLeaves)
                 this.auth.authData.userRootNamespace
               }}
               @class="namespace-link is-current button is-transparent icon"
             >
-              <Chevron @direction="left" @class="has-text-grey" />
+              <div class="is-flex-align-baseline">
```

Comment:
Is the div necessary here? Could this class be added directly to the button component?

Reply:
Yeah it's required. Otherwise it bumps the text out of the frame.